Article/Article.js

```diff
@@ -1,29 +1,45 @@
-// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoisted you will need to start your code at the bottom 
+//of the page.  
+//Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article=article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton=this.article.querySelector(".expandButton");
+    // Using your expandButton reference, update the text on your 
+    //expandButton to say "expand"
+    this.expandButton.textContent="expand";
     
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
+    // Set a click handler on the expandButton reference, calling the 
+    //expandArticle method.
+   this.expandButton.addEventListener("click",()=>this.expandArticle());
     
   }
 
   expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    // Using our reference to the domElement, toggle a class to expand or 
+    //hide the article.
+    this.expandButton.classList.toggle("article-open");
+    this.article.classList.toggle("article-open");
+    this.expandButton.classList.toggle("expandButton");
```

Comment:
Not sure why you included them, but lines 24 and 26 aren't needed here (in fact, those will break the appearance and functionality of your expand button after clicking it once).